template/Makefile.in

```diff
@@ -5,8 +5,7 @@ NULLCMD = @NULLCMD@
 silence = no # yes/no
 yes_silence = $(silence:no=)
 no_silence = $(silence:yes=)
-n=$(NULLCMD)
-ECHO1 = $(V:1=$n)
+ECHO1 = $(V:1=0)
```

Comment:
If this PR's function is approved, this line can be changed more.

```
ECHO1 = $(V:1=0)
```

Maybe it means if `V` is `1`, the `ECHO1` is set as `0`, and if `V` is `0` (not equal `1`), `ECHO1` is set as `V` (= `0`). It might be better for the line to be changed as follows.

```
ECHO1 = 0
```